.buildkite/scripts/package.sh

```diff
@@ -18,18 +18,11 @@ echo "--- run release-ci"
 export CSC_IDENTITY_AUTO_DISCOVERY=false
 # Disable notarize, see scripts/notarize.js
 export SKIP_NOTARIZATION=true
-npm run release-ci
+npm run release-ci | tee package.log
 
 # Store unsigned artifacts
 if [ -n "$BUILDKITE" ] ; then
     echo "--- Upload artifacts"
     mv dist artifacts-to-sign
-    # (only *nix)
-    buildkite-agent artifact upload "artifacts-to-sign/*.deb;artifacts-to-sign/*.dmg;artifacts-to-sign/*.zip"
-
-    # (only windows)
-    buildkite-agent artifact upload "artifacts-to-sign/*.exe"
-
-    # (only mac)
-    buildkite-agent artifact upload "artifacts-to-sign/*.dmg"
+    buildkite-agent artifact upload "artifacts-to-sign/*.*"
```

Comment:
See 

<img width="535" alt="image" src="https://github.com/user-attachments/assets/ca12b370-c9bd-430f-9b54-e4df11fc9761" />

```suggestion
    # We cannot upload artifacts-to-sign/*.* since it contains some generated files builder-debug, .blockmap and so on
    # (only *nix)
    buildkite-agent artifact upload "artifacts-to-sign/*.deb;artifacts-to-sign/*.dmg;artifacts-to-sign/*.zip"

    # (only windows)
    buildkite-agent artifact upload "artifacts-to-sign/*.exe"

    # (only mac)
    buildkite-agent artifact upload "artifacts-to-sign/*.dmg"
```